### PR TITLE
Renaming RUNNING to IN-PROGRESS within the ForEachItem execution #3453

### LINKED ISSUE #3453: Rename the RUNNING to IN-PROGRESS within the ForEachItem execution visualisation to clarify it covers all states in progress rather than only RUNNING
### Describe the issue

The subflows triggered by a ForEachItem are all indicated in "Running" even if they are in "Queued" (only for these 2 status)

### Environment

- Kestra Version: 0.15.6
- Operating System (OS/Docker/Kubernetes): OS

### PR DESCRIPTION
**Pull request by iamviqar**
### What changes are being made and why?

This closes #3453 
---

### How the changes have been QAed?

Validated on local setup, please refer the screenshot below 

Screenshot from 2024-10-15 16-25-15

---

### CODE REVIEW

**Comment by MilosPaunovic** [comment]
@Skraye Could you check this one to make sure it won't break anything, as it is confusing me what are we trying to achieve here and the way to get there.

File: ui/src/components/executions/ForEachStatus.vue [hunk 1]
```diff
@@ -79,6 +79,13 @@
             capitalizeFirstLetter(str) {
                 return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
             },
+            getStateToBeDisplayed(str){
+                if(str === State.RUNNING){
+                    return State.INPROGRESS;
```
Comment (Skraye) [inline ui/src/components/executions/ForEachStatus.vue]: Please replace by only  `IN-PROGRESS`, not using the state

File: ui/src/utils/state.js [hunk 2]
```diff
@@ -49,6 +49,15 @@ const STATE = Object.freeze({
...
     },
+    INPROGRESS: {
+        name: "INPROGRESS",
+        color: "#8405FF",
+        colorClass: "purple",
+        icon: PlayCircle,
+        isRunning: true,
+        isKillable: true,
+        isFailed: false,
+    },
```
Comment (Skraye) [inline ui/src/utils/state.js]: We wont add this as a real state for now, so you can remove it

File: ui/src/utils/state.js [hunk 3]
```diff
@@ -158,6 +167,10 @@ export default class State {
         return STATE.RUNNING.name;
     }
 
+    static get INPROGRESS() {
+        return STATE.INPROGRESS.name;
+    }
+
```
Comment (Skraye) [inline ui/src/utils/state.js]: Can also be removed

**Review by Skraye — CHANGES_REQUESTED** [review]
Good job! Even more than what we asked! 
Just some small changes and we're good to go :rocket:

**Comment by iamviqar** [comment]
@Skraye I've addressed your review comments, please re-review . Thanks

**Review by Skraye — APPROVED** [review]
LGTM! Nice contribution :medal_sports:

**Pull request merged**